core-services/prow/02_config/open-cluster-management-io/_prowconfig.yaml

```diff
@@ -1,6 +1,6 @@
 tide:
   merge_method:
-    open-cluster-management-io: squash
+    open-cluster-management-io: rebase
```

Comment:
We probably can't do this at the org level. I have doubts it'd be approved. We can override it per-repo for now and open a separate PR to propose it for the org (that way we're not blocked also).